src/user/password.ts

```diff
@@ -0,0 +1,57 @@
+import { CryptoUtils, EncoderUtils } from "../cryptography";
+import { TypeUtils, ValidatorUtils } from "../utils";
+import { PasswordOptions } from "./core";
+
+const defaultOptions = {
+  passwordValidator: ValidatorUtils.verifyStrongPassword,
+  iterations: 100,
+  keySize: 512 / 32,
+  salt: CryptoUtils.randomBytes(16),
```

Comment:
since we use a random salt, the hashing password will be different on each run?
when users come back with the same pwd but different salt, a new hash will be produced

this hashing pwd is used for AES utils to serialize/deserialize
I don't think it works

if it should work, we would need a test to cover this case

Reply:
The app need to store the salt and hashing options ( iterations, keySize), then reuse them later. 
I'll update the use-case.
https://github.com/CasperDash/casper-storage/pull/18/files#diff-909b4970c93b132525665a970642352afaf7e7f622f93f2a12b38ed3dc73ce55R243

Reply:
updated unit test and usage in readme